### ISSUE TITLE
Characters not allowed then rendering

### ISSUE THREAD

**Issue by andrefpoliveira** [sinaatalay/RenderCV#19]
**Describe the bug**
The package allows me to create a file with the name "André Oliveira". However, when trying to render it throws the following message:
`The input file contains a character that cannot be decoded with UTF-8 (b'\xe9')`

**To Reproduce**
```
rendercv new "André Oliveira"
rendercv render André_Oliveira_CV.yaml
```

**Screenshots**
https://imgur.com/a/MjjdaGf

**Comment by sinaatalay** [comment]
Thank you for finding this! The fix will be released with v1.2 (in a couple of days).

**Comment by sinaatalay** [comment]
This issue should have been solved with the new release. If you confirm, we can close the issue.
